apps/molgenis-components/src/components/account/Interfaces.ts

```diff
@@ -1,4 +1,4 @@
-import { ISetting } from "../../Interfaces/ISetting";
+import type { ISetting } from "meta-data-utils/dist/types";
```

Comment:
I would have expected you could import from "meta-data-utils", why is that different for 'type'?

Reply:
i think because there not in the index.js , sure about what the best options is here , non a fan of having to redeclare everything in a export file